### PR TITLE
Add VAPOR 2.6.0

### PR DESCRIPTION
**Pull request by leifdenby**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/vapor.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'vapor' do
+  version '2.6.0'
+  sha256 '0608952d436550bb22d692cef43087c2ed3d96fa09b5b13eaaedf24143ddd5c6'
+
+  url 'https://www.vapor.ucar.edu/sites/default/files/attachments/vapor-2.6.0-Darwin_x86_64.dmg'
```
Comment (commitay) [inline Casks/vapor.rb]: `"https://www.vapor.ucar.edu/sites/default/files/attachments/vapor-#{version}-Darwin_x86_64.dmg"`

File: Casks/vapor.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  name 'VAPOR is the Visualization and Analysis Platform for Ocean, Atmosphere, and Solar Researchers'
```
Comment (commitay) [inline Casks/vapor.rb]: Just `VAPOR` will do here, `name` is not meant to be a description.

File: Casks/vapor.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  homepage 'https://www.vapor.ucar.edu/'
+
+  pkg 'vapor.pkg'
+end
```
Comment (commitay) [inline Casks/vapor.rb]: Please add `uninstall pkgutil` 

[uninstall Key pkgutil](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/uninstall.md#uninstall-key-pkgutil)

Example:
https://github.com/caskroom/homebrew-cask/blob/7ef0a7a277d022c0d93ea6371a10688ed4408d4b/Casks/qgis.rb#L12-L14

**Review by commitay — DISMISSED** [review]

**Comment by miccal** [comment]
Thank you for your contribution.

**Pull request merged**